content/guides/authentication/jwt/without-sdk.md

```diff
@@ -16,52 +16,51 @@ alias_paths:
 
 # JWT without SDKs
 
-If you are not ready to use any of the official Box SDKs, or an SDK is not
-available in your language of choice, it is totally possible to use the Box APIs
-without them.
+This guide takes you through JWT authentication without using a Box SDK.
+JWT does not require end-user interaction and is designed to authenticate
+directly with the Box API. 
 
-This guide will take you through user authentication using JWT without the use
-of the Box SDKs. JWT authentication is designed for working directly with the
-Box API without requiring a user to redirect through Box to authorize your
-application.
+There are two ways you can verify an application's permissions:
 
-## Overview
+- using a public and private key pair
```

Comment:
Might want to link these two items to their sections on the same page: #using-a-key-pair & #client-credentials-grant